### ISSUE TITLE
Re Get RVT Info Error in Revit 2016

### ISSUE THREAD

**Issue by Robbo1234** [eirannejad/pyRevit#453]
### Issue Description
I clicked on Project: Get RVT Info and selected a Revit model and I got this error, refer to screen shot.
p.s. The error is probably because I've tested in Revit 2016 Build 16.0.1185.0 because I getting a similar error with PyRevit Pacakage and Tags.

### Environment

```
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive)
Name: "master" | Deploy: "basepublic" | Path: "C:\Program Files\pyRevit-Master"
==> Attachments
Autodesk Revit 2014 First Customer Ship | Clone: "master" | Engine: "277"
Autodesk Revit 2016 Update 7 for R2 | Clone: "master" | Engine: "277"
Autodesk Revit 2017.2.3 | Clone: "master" | Engine: "277"
Autodesk Revit 2018.3.1 | Clone: "master" | Engine: "277"
Autodesk Revit 2019.1 | Clone: "master" | Engine: "277"
==> Installed UI Extensions
==> Installed Library Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2014 First Customer Ship | Version: 13.3.8151 | Language: 1033 |
Path: "C:\Program Files\Autodesk\Revit 2014\"
Autodesk Revit 2016 Update 7 for R2 | Version: 16.0.1185.0 | Language: 1033 | Pa
th: "C:\Program Files\Autodesk\Revit 2016\"
Autodesk Revit 2017.2.3 | Version: 17.0.1150.0 | Language: 1033 | Path: "C:\Prog
ram Files\Autodesk\Revit 2017\"
Autodesk Revit 2018.3.1 | Version: 18.3.1.2 | Language: 1033 | Path: "C:\Program
 Files\Autodesk\Revit 2018\"
Autodesk Revit 2019.1 | Version: 19.1.0.112 | Language: 1033 | Path: "C:\Program
 Files\Autodesk\Revit 2019\"
==> Running Revit Instances
PID: 12484 | Autodesk Revit 2016 Update 7 for R2 | Version: 16.0.1185.0 | Langua
ge: 0 | Path: "C:\Program Files\Autodesk\Revit 2016"
```

![image](https://user-images.githubusercontent.com/8776947/50169962-ac6bf280-02e6-11e9-9fca-822d2be92c57.png)

**Comment by Robbo1234** [comment]
FYI: 
I got the same error for Project: Get RVT Info in Revit 2017.2.3

**Comment by eirannejad** [comment]
Okay seems like this is a DLL load issue. Will work on it

**Comment by Robbo1234** [comment]
Thanks Ehsan

**Comment by Robbo1234** [comment]
Thanks Ehsan

**Comment by eirannejad** [comment]
Resolved and will be pushed with 4.6.9 this weekend

**Comment by Robbo1234** [comment]
Thanks Eshan

**Comment by Robbo1234** [comment]
Installed pyRevit 4.6.17 and I got the error when I clicked on pyRevit Get RVT Info

Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\rob.cross>pyrevit env
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive)
master | Deploy: "basepublic" | Branch: "master" | Version: "4.6.17" | Path: "C:
\Users\rob.cross\AppData\Roaming\pyRevit-Master"
==> Attachments
Autodesk Revit 2016 Update 7 for R2 | Clone: "master" | Engine: "277"
Autodesk Revit 2017.2.3 | Clone: "master" | Engine: "277"
Autodesk Revit 2018.3.1 | Clone: "master" | Engine: "277"
Autodesk Revit 2019.2 (Update) | Clone: "master" | Engine: "277"
==> Installed UI Extensions
==> Installed Library Extensions
==> Default Extension Search Path
C:\Users\rob.cross\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2016 Update 7 for R2 | Version: 16.0.1185.0 | Language: 1033 | Pa
th: "C:\Program Files\Autodesk\Revit 2016\"
Autodesk Revit 2017.2.3 | Version: 17.0.1150.0 | Language: 1033 | Path: "C:\Prog
ram Files\Autodesk\Revit 2017\"
Autodesk Revit 2018.3.1 | Version: 18.3.1.2 | Language: 1033 | Path: "C:\Program
 Files\Autodesk\Revit 2018\"
Autodesk Revit 2019.2 (Update) | Version: 19.2.0.65 | Language: 1033 | Path: "C:
\Program Files\Autodesk\Revit 2019\"
==> Running Revit Instances
PID: 19808 | Autodesk Revit 2017.2.3 | Version: 17.0.1150.0 | Language: 0 | Path
: "C:\Program Files\Autodesk\Revit 2017"
==> User Environment
Microsoft Windows 7 [Version 6.1.7601]
Executing User: COADY\rob.cross
Active User: COADY\rob.cross
Adming Access: Yes
%APPDATA%: "C:\Users\rob.cross\AppData\Roaming"
Latest Installed .Net Framework: "4.7.2"
No .Net Target Packs are installed.
pyRevit CLI 0.9.0.0

C:\Users\rob.cross>^A


IronPython Traceback:
Traceback (most recent call last):
 File "C:\Users\rob.cross\AppData\Roaming\pyRevit-Master\extensions\pyRevitTools.extension\pyRevit.tab\Project.panel\ptools2.stack3\Get RVT Info.pushbutton\script.py", line 15, in 
IOError: System.IO.FileLoadException: Could not load file or assembly 'OpenMcdf, Version=2.2.1.2, Culture=neutral, PublicKeyToken=fdbb1629d7c00800' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
File name: 'OpenMcdf, Version=2.2.1.2, Culture=neutral, PublicKeyToken=fdbb1629d7c00800' ---> System.IO.FileLoadException: A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
 at Microsoft.Scripting.Interpreter.NewInstruction.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run4[T0,T1,T2,T3,TRet](T0 arg0, T1 arg1, T2 arg2, T3 arg3)
 at System.Dynamic.UpdateDelegates.UpdateAndExecute3[T0,T1,T2,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2)
 at Microsoft.Scripting.Interpreter.DynamicInstruction`4.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run2[T0,T1,TRet](T0 arg0, T1 arg1)
 at IronPython.Compiler.PythonScriptCode.RunWorker(CodeContext ctx)
 at PyRevitBaseClasses.ScriptExecutor.ExecuteScript(PyRevitCommandRuntime& pyrvtCmd)




Script Executor Traceback:
System.IO.FileLoadException: Could not load file or assembly 'OpenMcdf, Version=2.2.1.2, Culture=neutral, PublicKeyToken=fdbb1629d7c00800' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
File name: 'OpenMcdf, Version=2.2.1.2, Culture=neutral, PublicKeyToken=fdbb1629d7c00800' ---> System.IO.FileLoadException: A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
 at Microsoft.Scripting.Interpreter.NewInstruction.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run4[T0,T1,T2,T3,TRet](T0 arg0, T1 arg1, T2 arg2, T3 arg3)
 at System.Dynamic.UpdateDelegates.UpdateAndExecute3[T0,T1,T2,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2)
 at Microsoft.Scripting.Interpreter.DynamicInstruction`4.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run2[T0,T1,TRet](T0 arg0, T1 arg1)
 at IronPython.Compiler.PythonScriptCode.RunWorker(CodeContext ctx)
 at PyRevitBaseClasses.ScriptExecutor.ExecuteScript(PyRevitCommandRuntime& pyrvtCmd)

![image](https://user-images.githubusercontent.com/8776947/53725044-16feea00-3e63-11e9-8a31-99ab4ba4c670.png)
